packages/snap/snap.config.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  cliOptions: {
+    src: './build/index.js',
```

Comment:
since snaps-cli supports typescript now, maybe we don't need to compile manually anymore?